From 2a1ae7b6f34a96ea25be007ac9d91e57e9d32284 Mon Sep 17 00:00:00 2001
From: David Seifert <soap@gentoo.org>
Date: Wed, 26 Feb 2020 13:24:17 +0100
Subject: [PATCH 1/1] Fix building against GCC 10

* GCC 10 defaults to `-fno-common`, which makes C behave
  more like C++ in that you can only ever have one definition
  of an object per executable.
---
 rle.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/rle.h b/rle.h
index 0d59484..4f8946d 100644
--- a/rle.h
+++ b/rle.h
@@ -30,7 +30,7 @@ extern "C" {
  *** 43+3 codec ***
  ******************/
 
-const uint8_t rle_auxtab[8];
+extern const uint8_t rle_auxtab[8];
 
 #define RLE_MIN_SPACE 18
 #define rle_nptr(block) ((uint16_t*)(block))
-- 
2.33.1

